### ISSUE TITLE
Readurl is not working in VS 

### ISSUE THREAD

**Issue by dkdarshu45** [mulesoft-labs/data-weave-language-server#39]
I was trying to execute readurl function in the extension with and without classpath but somehow both are not working...
While reading using readurl ( classpath :// )
I kept file inside resources folder and without classpath I tried keeping file in my c directory.

Could you please try and confirm this is working for you .
In both cases I get file not found
I am checking on windows.

**Comment by uip-robot-zz** [comment]
Git2Gus App is installed but the `.git2gus/config.json` doesn't exist.

**Comment by machaval** [comment]
Hi @dkdarshu45 can you please attach an example app that reproduces the issue?

**Comment by svacas** [comment]
try adding `src/main/resources` as a resource in your pom.xml (till this is fixed in a new release)
```
        <resources>
            <resource>
                <directory>src/main/dw</directory>
            </resource>
            <resource>
                <directory>src/main/resources</directory>
            </resource>
        </resources>
```
If you run a test that should run maven for you, if you want to run a preview you'll have to run maven from the terminal before that
```
mvn process-resources
```

**Comment by dkdarshu45** [comment]
Thanks @svacas . It's working fine with the workaround you mentioned. Only thing is we run mvn clean  then we again need to run mvn process-resources for the preview feature to work as expected

**Comment by svacas** [comment]
right, that's how maven works, you can run both together to get a clean classpath generated
```
mvn clean process-resources
```

**Comment by dkdarshu45** [comment]
@svacas 
To add to that if I want to change content of the file read on classpath preview feature is not updated each time we need to run mvn clean as you said. if I update the file contents directly inside target folder preview feature works fine. But it's not very intuitive.
I wanted to use this plugin similar to playground (dynamically updating payload ).
How do you suggest should I go about it in this case

**Comment by machaval** [comment]
This is a bug and it should work as you said it should not be required to do mvn clean  for this to be reflected in the preview. Thanks we will try to address this on next release
